src/metabase/metabot.clj

```diff
@@ -9,7 +9,8 @@
 (defsetting metabot-enabled
```

Comment:
should this be deprecated?

Reply:
Yeah, I'll mark it as deprecated as well. I'll probably be putting up a subsequent PR that deletes all the metabot code anyway, so it might not matter, but doesn't hurt.